src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,32 @@
+.section-title {
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: -1.5px;
+  margin: 0;
+
+  @include onTable {
+    font-size: 40px;
+  }
+
+  @include onDesktop {
+    font-size: 48px;
+  }
+}
+
+.section-title__position {
```

Comment:
The class '.section-title__position' seems like it might be better as a modifier of '.section-title' rather than a separate element, based on BEM methodology. Consider renaming it to '.section-title--position' if it's a modifier, or if it's a separate element, make sure it's a child of the '.section-title' element in your HTML.